packages/user-operation-controller/src/helpers/Bundler.ts

```diff
@@ -91,12 +95,10 @@ export class Bundler {
 
     log('Sent user operation', hash);
 
-    return hash;
+    return hash as string;
   }
 
-  // TODO: Replace `any` with type
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async #query(method: string, params: any[]): Promise<any> {
+  async #query(method: string, params: unknown[]): Promise<unknown> {
```

Comment:
Should we templetize this method so we don't need to cast the return value?
```suggestion
  async #query<Response>(method: string, params: unknown[]): Promise<Response> {
```

Reply:
Done.